setup.py

```diff
@@ -42,5 +42,8 @@ def get_version(*file_paths):
         "lms.djangoapp": [
             "seb_openedx = seb_openedx.apps:SebOpenEdxConfig",  # retain
         ],
+        "cms.djangoapp": [
```

Comment:
Last time we tried to have two endpoints with the same entry point, it failed. Is this working alright?

Reply:
Yeah It's working alright (but cms part is going to be deleted so it doesn't matter much)